### PR TITLE
Update playbooks to fix bug in beaker destroy

### PR DESCRIPTION
**Pull request by samvarankashyap**
Summary:
A bug was recently introduced in linchpin beaker making it unable to provision resource inventory and destroy them.
Committer: samvaran kashyap <samvaran.kashyap@gmail.com>

fixes: #1179 1179

### CODE REVIEW

**Comment by samvarankashyap** [comment]
@14rcole 
Needs review

File: setup.py [hunk 1]
```diff
@@ -35,7 +35,7 @@
         [console_scripts]
         linchpin=linchpin.shell:runcli
     ''',
-    tests_require=["pytest", "nose", "mock", "coverage", "flake8"],
+    tests_require=["pytest<=4.4.0", "nose", "mock", "coverage", "flake8"],
```
Comment (14rcole) [inline setup.py]: What changed in newer versions of pytest?

Comment (samvarankashyap) [reply setup.py]: Didn't actually check on that. But its currently failing on on latest version.

**Comment by samvarankashyap** [comment]
[merge]

**Pull request merged**